src/lib/components/Spinner.svelte

```diff
@@ -18,8 +18,8 @@
 		height: 50px;
 		border-radius: 50%;
 		position: fixed;
-		top: 50%;
-		left: 50%;
+		top: calc(50% - 25px);
+		left: calc(50% - 25px);
```

Comment:
While this works, the more common way of doing it would be adding a `transform: translate(-50%, -50%)` style to the class. If the size of the spinner ever changed, we would only need to edit 2 values instead of 4.

Reply:
Understood! Will try and implement it!

Reply:
![image](https://github.com/revanced/revanced-website/assets/84234554/bd3fa902-7045-4c92-95f9-06eb617d7e55)

I've certainly fixed it that way, but i'll need your review for this

![image](https://github.com/revanced/revanced-website/assets/84234554/e9c78e84-b3d7-4c1a-ae59-32ce969066ac)

I've added a container over the spinner, and used `.spinner` instead of `::before` so that it can access the full width of the container. Is this good of a fix? 

Please let me know, your thoughts.

Reply:
I've rebased and pushed the code, please review!